src/test/java/seedu/address/model/person/OccurrenceTest.java

```diff
@@ -25,9 +25,24 @@ public void getNext_validOccurrence_success() {
     public void isValidOccurrence() {
         // invalid occurrence
         assertFalse(Occurrence.isValidOccurrence(-1));
+        assertFalse(Occurrence.isValidOccurrence("-1"));
+        assertFalse(Occurrence.isValidOccurrence(-100));
+        assertFalse(Occurrence.isValidOccurrence("-100"));
         assertFalse(Occurrence.isValidOccurrence(0));
+        assertFalse(Occurrence.isValidOccurrence("0"));
+        assertFalse(Occurrence.isValidOccurrence(1000)); // occurrence should be less than or equals 999
+        assertFalse(Occurrence.isValidOccurrence("1000"));
+        // large integers
```

Comment:
might want to add some more comments for the grouping of tests.

Reply:
okays!